openwisp_controller/config/base/multitenancy.py

```diff
@@ -33,3 +33,36 @@ class Meta:
 
     def __str__(self):
         return self.organization.name
+
+
+class AbstractOrganizationAllowedDevices(UUIDModel):
```

Comment:
The name here sounds too specific, I would instead try to use something more generic so that in the future we can add other limitations, for example we could limit the number of VPN servers, the number of Templates, etc.

So I would call this, `AbstractOrganizationLimits` we could use the same name also in other modules so we have to make sure the `verbose_name` is a bit more specific.